src/Sentry/Laravel/Features/CacheIntegration.php

```diff
@@ -81,7 +82,16 @@ public function handleRedisCommand(RedisEvents\CommandExecuted $event): void
 
         $context = new SpanContext();
         $context->setOp('db.redis');
-        $context->setDescription(strtoupper($event->command) . ' ' . ($event->parameters[0] ?? null));
+
+        $keyForDescription = '';
+
+        // If the first parameter is a string and does not contain a newline we use it as the description since it's most likely a key
+        // This is not a perfect solution but it's the best we can do without understanding the command that was executed
+        if (!empty($event->parameters[0]) && is_string($event->parameters[0]) && !Str::contains($event->parameters[0], "\n")) {
+            $keyForDescription = $event->parameters[0];
+        }
+
+        $context->setDescription(rtrim(strtoupper($event->command) . ' ' . $keyForDescription));
```

Comment:
Might add a blank ` ` at the end, but not a biggie.

Reply:
Have an `rtrim` to prevent that 😉